src/streaming/text/TextTracks.js

```diff
@@ -543,21 +543,27 @@ function TextTracks() {
         }
     }
 
-    function deleteTrackCues(track) {
+    function cueInRange(cue, start, end) {
+        return (cue.startTime >= start || isNaN(start)) && (cue.endTime <= end || isNaN(end));
```

Comment:
I didn't test this code but I think it's better to check isNaN before the time value compared to cue time value....Do you agree?

Reply:
Sure!